src/phoenix/trace/fixtures.py

```diff
@@ -139,6 +139,29 @@ class TracesFixture:
     ),
 )
 
+demo_toolcalling_fixture = TracesFixture(
+    name="demo_toolcalling",
```

Comment:
```suggestion
demo_tool_calling_fixture = TracesFixture(
    name="demo_tool_calling",
```

Non-blocking suggestion